notebooks/14-combine-data.md

```diff
@@ -107,16 +107,12 @@ date_var
 4. Load in and concatenate all individual GeoTIFFs
 
 ```{code-cell} ipython3
-moisture_index = xr.concat([xr.open_rasterio(file_name) for file_name in moisture_index_files], dim=date_var)
+moisture_index = xr.concat([xr.open_dataarray(file_name, engine="rasterio") for file_name in moisture_index_files], dim=date_var)
+moisture_index = (moisture_index - moisture_index.min())/(moisture_index.max() - moisture_index.min()) # rescale to 0-1 range for RGB plotting
```

Comment:
Do you know when this is exactly needed or when not?

Because when running this, I get exactly the same plot (and without warning) if Ieave out this rescaling in this case (although I have seen the warning in other notebooks)

Reply:
I think (would need to double check), when matplotlib receives 3 'layers' (RGB), the warning will appear; when only 1 layer, no warning is raised.

Reply:
Ah, yes, because if 3 layers, it tries to interpret it as R, G and B values, and so those can't be arbitrary values. 

But so in this case there is only a single band, and thus rescaling doesn't matter.